### DISCUSSION TITLE
What course do you use for side delivery rakes?

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#3483] [Q&A]
What course and pattern do you use for side delivery rakes? is there any type that works where you can then send a baler behind it by offsetting etc... curious as they drive me mad if I get them on a contract!

![fsScreen_2024_09_29_20_47_08](https://github.com/user-attachments/assets/95fb1340-2eb7-4899-bbf4-ead17af21117)

**Comment by Tensuko** [comment]
Start on center
Headland clockwise
Up/Down pattern

**Reply by coatsy35** [reply]
Trouble is up down pattern puts two 9m together which is bigger than a baler can handle. Do you use any offsets? Headland is worse as I can't find a way to allow a baler to follow it as it just rakes everything into one big row near to the centre as it goes around and around.

**Reply by Tensuko** [reply]
So when swath geta too big use other patterns :)
Just check how they would drive and then see where it should land.

**Comment by rohne83** [comment]
Best solution: buy your own windrower ;)
With spirals or lands (make sure to use a high number of rows per land) for field center you should be able to have many rows not put together with the next row. But could be hard to be avoided completely.
Or you go to the fruitTypes.xml of the map and decrease the grass yield to more realistic levels where two rows put together is even desired then.
